custom/conf/app.example.ini

```diff
@@ -1133,6 +1133,10 @@ PATH =
 ;;
 ;; Whether to enable a Service Worker to cache frontend assets
 ;USE_SERVICE_WORKER = false
+;;
+;; Whether to only show relevant repos on the explore page when no keyword is specified and default sorting is used.
+;; A repo is considered irrelevant if it's a fork or doesn't have set metadata (description, topic).
+;HIDE_IRRELEVANT_REPOS = false
```

Comment:
calling them irrelevant is maybe not the right thing? Just wondering if renaming it makes more sense, e.g. to
`ONLY_SHOW_RELEVANT_REPOS = false`
or
`EXPLORE_SHOW_ALL_REPOS = true`

Reply:
Thank you for your feedback: https://github.com/go-gitea/gitea/pull/19361/commits/aa8223ff54d3819536703b72ba657b3a97cc23ae the old naming indeed didn't really made any sense.

Reply:
Could it be an option on the page instead of a global setting option? I think we should avoid more and more global options. And some users may need it while some others may not.

Reply:
Option on the page? Should such choice be saved in the user's settings table? With the default setting being the one of the global option.

Reply:
I'm in favour of the current behaviour. It defines the default, but the user can avoid by clicking one the page to show all. I prefer this over another user setting which you can easily forget about.